ui/package.json

```diff
@@ -17,7 +17,7 @@
     "@types/react": "18.0.25",
     "@types/react-dom": "18.0.9",
     "coi-serviceworker": "^0.1.6",
-    "coinflip-executor-contract": "0.5.2",
+    "coinflip-executor-contract": "0.5.3",
```

Comment:
this version bump gives us a back door to reset the state to empty if we get out of sync again